hooks/get_current_login.py

```diff
@@ -46,7 +47,7 @@ def execute(self, **kwargs):
         :returns: A name that matches the user's ``login`` field in Shotgun.
         :rtype: str
         """
-        if sys.platform == "win32":
+        if sgsix.platform == "win32":
```

Comment:
sgsix.platform? I don't this defined here: https://github.com/shotgunsoftware/python-api/blob/master/shotgun_api3/lib/sgsix.py